### ISSUE TITLE
Whitney Grammar

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/COLOGNE#314]
I've made some progress on a display of Whitney's Sanskrit  Grammar.

Please see https://github.com/funderburkjim/WhitneyGrammar/issues/1 .  

Comments requested.

**Comment by funderburkjim** [comment]
Also, please take a look at Issues 2 and 3 under above.

**Comment by gasyoun** [comment]
> Comments requested.

Only after the karaka dependency step from Jim ))

**Comment by gasyoun** [comment]
**Question type:** content-enhancement (prototype display feedback request)

**Evidence found:**

Jim's [WhitneyGrammar repository](https://github.com/funderburkjim/WhitneyGrammar) (2020) proposed a page-by-page HTML display of Whitney's *Sanskrit Grammar* sourced from the Wikisource digitization. Three sub-issues were opened there:
- Issue 1: UI/display prototype review
- Issue 2: IAST normalization (Whitney uses āi/āu/ṅ/ç vs. current standards — 310 diphthong + 1168 palatal sibilant differences)
- Issue 3: Missing bold markup for Sanskrit-IAST in the Wikisource transcription

The repo was last committed 2020-11-12 and has not been developed further (3 open issues, 0 forks).

Since 2020 the work has advanced on a different axis: [WhitneyRoots](https://gasyoun.github.io/WhitneyRoots/) now provides a structured, crosslinked display of Whitney's *Roots* (not Grammar) with full §-paragraph citations mapped to each root — see [Whitney_Grammar_Citations.md](https://github.com/gasyoun/WhitneyRoots/blob/main/Whitney_Grammar_Citations.md) (935 roots × Grammar §§, with generic/specific/exception markers) and a 2-layer Whitney↔DCS↔MW crosswalk covering ~24 k triples.

**Assessment:** Partially answerable. The original prototype feedback loop (UI/IAST/bold) was never formally closed — the WhitneyGrammar repo stalled. However, the underlying concern (linking Grammar §§ to roots) has been addressed in WhitneyRoots via the Grammar Citations table and the root reader at https://gasyoun.github.io/WhitneyRoots/.

**Suggested next step:** Decide whether to (a) close this as superseded by the WhitneyRoots Grammar §-citation integration, or (b) reopen the WhitneyGrammar display project under the sanskrit-lexicon org with the IAST normalization issues (āi→ai, ç→ś, etc.) as the first concrete task.
